Libraries/Drei/views.lisp

```diff
@@ -81,37 +81,46 @@ If empty, tabs every TAB-WIDTH are assumed.")))
   (:documentation "Return the width of a space character on
 `stream' in device units (most likely pixels).")
   (:method ((stream extended-output-stream) (tabify tabify-mixin))
-    (maybe-update-recordings stream tabify)
-    (recorded-space-width tabify)))
+	   (maybe-update-recordings stream tabify)
+	   (with-accessors ((recorded-space-width recorded-space-width)) tabify
+			   recorded-space-width)))
 
 (defgeneric tab-width (stream tabify)
   (:documentation "Return the width of a tab character on
 `stream' in device units (most likely pixels).")
   (:method ((stream extended-output-stream) (tabify tabify-mixin))
-    (if (tab-space-count tabify)
-        (* (tab-space-count tabify) (space-width stream tabify))
-        (recorded-tab-width tabify))))
+	   (with-accessors ((tab-space-count tab-space-count) (recorded-tab-width recorded-tab-width) (use-tabs use-tabs)) tabify
+			   (if use-tabs
+			       recorded-tab-width
+				 (* (if tab-space-count tab-space-count 8) (space-width stream tabify))))))
```

Comment:
iden't here and above are awful, please fix that and explain in a commit why this change is necessary.